pkg/cloud/baremetal/actuators/machine/actuator_test.go

```diff
@@ -204,22 +192,6 @@ func TestChooseHost(t *testing.T) {
 			ExpectedHostName: host2.Name,
 			Config:           config,
 		},
-		{
-			Scenario: "should ignore discoveredHost and pick host2, which lacks a ConsumerRef",
-			Machine: machinev1beta1.Machine{
-				ObjectMeta: metav1.ObjectMeta{
-					Name:      "machine1",
-					Namespace: "myns",
-				},
-				TypeMeta: metav1.TypeMeta{
-					Kind:       "Machine",
-					APIVersion: machinev1beta1.SchemeGroupVersion.String(),
-				},
-			},
-			Hosts:            []runtime.Object{&discoveredHost, &host2, &host1},
-			ExpectedHostName: host2.Name,
-			Config:           config,
-		},
```

Comment:
This test change seems to be related to the rebase, so it'd be clearer if it was part of that commit IMO (it makes sense when you view the PR diff, but not if you look at each commit individually, because this test is using the still-valid `StateRegistering
`)

Reply:
@hardys - `StateRegistering` definitely still exists hence why I left it in actuator.go, but `StateRegistrationError` was removed into a different code path.

Reply:
I've added them back in, as I think they do still run just fine.